modules/tinymce/src/core/test/ts/browser/commands/LineHeightTest.ts

```diff
@@ -0,0 +1,64 @@
+import { Log, Pipeline, Step } from '@ephox/agar';
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { TinyApis, TinyLoader } from '@ephox/mcagar';
+import Editor from 'tinymce/core/api/Editor';
+
+UnitTest.asyncTest('LineHeightTest', (success, failure) => {
```

Comment:
The test name should be something like `browser.tinymce.core.commands.LineHeightTest`

- [ ] Fix incorrect test name